lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -91,6 +91,14 @@ function _reportback_item_resource_definition() {
             'source' => array('param' => 'load_user'),
             'default value' => FALSE,
           ],
+          [
+            'name' => 'reaction_user',
```

Comment:
Do you think this makes more sense as `user` or `as_user` since it really sets the user for the entire request?